scripts/delete_trailing_whitespace.sh

```diff
@@ -18,7 +18,9 @@ if [ ! -d "$REPO_DIR" ]; then
   echo "$REPO_DIR directory does not exist";
 else
   while read -rd '' fname; do
-    if grep -q "[[:space:]]$" "$fname"; then
+    if [[ $fname =~ "tex" ]]; then
+      continue
+    elif grep -q "[[:space:]]$" "$fname"; then
```

Comment:
Instead of making this change, would it work to instead to change line 30 below from
`done < <( find "$REPO_DIR" -path "*/contrib" -prune -o -path "*/libmesh" -prune -o \( -name "*.[Chi]" -o -name "*.py" -o -name "*.tex" \) -type f -print0)`
to
`done < <( find "$REPO_DIR" -path "*/contrib" -prune -o -path "*/libmesh" -prune -o \( -name "*.[Chi]" -o -name "*.py" \) -type f -print0)`?
This, way, it shouldn't match any `.tex` files.

Reply:
oh nice